src/transform3d.rs

```diff
@@ -177,6 +177,18 @@ where T: Copy + Clone +
         )
     }
 
+    /// Check whether shapes on the XY plane with Z pointing towards the
+    /// screen transformed by this matrix would be facing back.
+    pub fn is_backface_visible(&self) -> bool {
+        // inverse().m33 < 0;
```

Comment:
I refer to gecko's implementation.  https://searchfox.org/mozilla-central/source/gfx/2d/Matrix.h#1392-1400